src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -771,24 +771,25 @@ def uploadRSEToMongoDB(self, rse, fullRSEToDB=False, overwrite=True):
             raise MSUnmergedPlineExit(msg)
         return rse
 
-    def getStatsFromMongoDB(self, **kwargs):
+    def getStatsFromMongoDB(self, detail=False, **kwargs):
         """
         Auxiliary method to serve the APIs of the REST calls for the service.
         Implements various queries to MongoDB based on the parameters passed.
         :param rse=rseName:      String representing the RSE name to query the database for
-        :param rse_type=rseType: String representing the RSE type to query the database for
```

Comment:
and `detail` here could be part of the kwargs as well. 
Regardless of what you select to implement here, please update the docstring.

Reply:
I'd prefer to keep it with a default value here, so we protect ourselves of unneeded huge dumps in the browser of enormous RSE's like CERN or FNAL - a mistake I have already done several times myself while testing it. 
I'll update the docstring.